### DISCUSSION TITLE
Prettier - disable new line on specific attributes in JSX (e.g `style` props)

### DISCUSSION THREAD

**Discussion by flixyudh** [prettier/prettier#13767] [Q&A] [answered]
I'm using `Prettier` in my `React Native` project, and i keep use `inline style` for styling component, but when i declare long style that exceed `printWidth` my style being formatted.

example:
```js
//input
<Text style={{fontSize: 34, fontFamily: 'Custom Font', letterSpacing: 1.5, marginBottom: 8, color: colors.primary, textAlign: 'center'}}>
  My Title
</Text>

//prettier formatted
<Text
  style={{
    fontSize: 34,
    fontFamily: 'Custom Font',
    letterSpacing: 1.5,
    marginBottom: 8,
    color: colors.primary,
    textAlign: 'center',
  }}
>
  My Title
</Text>
```

I want to keep my `style` props into single line, even when i declare more props something like this

```js
//input
<Text numberOfLines={3} style={{fontSize: 34, fontFamily: 'Custom Font', letterSpacing: 1.5, marginBottom: 8, color: colors.primary, textAlign: 'center'}}>
  My Title
</Text>

//expected formatted output
<Text
  numberOfLines={3}
  style={{fontSize: 34, fontFamily: 'Custom Font', letterSpacing: 1.5, marginBottom: 8, color: colors.primary, textAlign: 'center'}}
>
  My Title
</Text>
```

is there some way to setup prettier disable new line for specific attributes?

**Comment by duailibe** [comment, marked as answer]
> is there some way to setup prettier disable new line for specific attributes?

No, there isn't. The only thing you can do is add a`// prettier-ignore` comment right before the line, but you'll also lose any other formatting.
